### ISSUE TITLE
A nest in itemization become canceled.

### ISSUE THREAD

**Issue by harumo11** [lawmurray/doxide#45]
Hello. Thanks for good tool.

I wrote following comments using itemization in header file.
```c++
- vector2 : time parameters of filter
    - sampling period [s]
    - time constant [s]
````

When I run `doxide build`, I got following result in converted markdown file.
```
- vector2 : time parameters of filter
- sampling period [s]  <-- nest is deleted!
- time constant [s]      <-- nest is deleted!
```

Did I write something wrong?
If you know a solution, please let me know.

Thanks in advance.

**Comment by lawmurray** [comment]
Thanks for the report @harumo11. I'm looking into this.

**Comment by lawmurray** [comment]
@harumo11 What sort of documentation comments are you using here? e.g. `///` or `/** ... */` style?

**Comment by lawmurray** [comment]
I'm doing some work on the release pipeline, but this should be available in a proper release within a few days. In the meantime it is available if installing from source.

**Comment by harumo11** [comment]
> @harumo11 What sort of documentation comments are you using here? e.g. `///` or `/** ... */` style?

I used `/** ... */` style comment.

**Comment by harumo11** [comment]
> I'm doing some work on the release pipeline, but this should be available in a proper release within a few days. In the meantime it is available if installing from source.

Thank you for your super quick response!!
I'll download the source code and try it out.

**Comment by lawmurray** [comment]
Thanks for the update @harumo11 ! And for everyone else, the fix is included in the latest v0.4.6, download available from the site.
